MODULE.bazel

```diff
@@ -1,6 +1,6 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "triangulator", version = "0.0")
+
+bazel_dep(name = "catch2")
```

Comment:
these two lines could be `bazel_dep(name = "catch2", version = "3.6.0")` (with a bit outdated version)

Reply:
done